package.json

```
@@ -144,6 +144,12 @@
           "default": "",
           "markdownDescription": "Pass additional arguments to the language server."
         },
+        "haskell.envVars": {
+          "scope": "resource",
+          "type": "object",
+          "default": {},
+          "markdownDescription": "Define environment variables for compilation/execution."
```

We are passing the env vars for the lsp server (haskell-language-server) at runtime so not sure if `for compilation/execution` would be accurate 
What about 
```suggestion
          "markdownDescription": "Define environment variables for the language server."
```